lib/errorDefinitions.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache License. See License in the project root for license information.
+
+/**
+ * Defines the error that occurs when an operation timeout occurs.
+ */
+export class OperationTimeoutError extends Error {
+  /**
+   * Describes the name of the error.
+   */
+  readonly name: string = "OperationTimeoutError";
+
+  constructor(message: string) {
+    super(message);
+  }
+}
+
+/**
+ * Defines the error that occurs when the Sender does not have enough credit.
+ */
+export class InsufficientCreditError extends Error {
```

Comment:
the new errors are not captured in the changelog and not exported in index.ts

Reply:
I intentionally did not export them. However, it just struck me if the user would want to check the Error or may be extend the error they will need the Error object. Will export them and update the changelog.